auslib/admin/base.py

```diff
@@ -77,6 +91,7 @@ def add_security_headers(response):
 # hosted at "/api", which is stripped away by the web server before we see
 # these requests.
 app.add_url_rule("/csrf_token", view_func=CSRFView.as_view("csrf"))
+app.add_url_rule("/current_user", view_func=CurrentUserView.as_view("current_user"))
```

Comment:
/users/:id actually doesn't exist (though maybe it should...). I do like the idea of having this endpoint somewhere in the /users namespace, though I'm not sure I like having a magic string there. I think I'll give this a try and see how it looks/feels.